generator/src/test/java/com/scottlogic/deg/generator/cucumber/If.feature

```diff
@@ -2435,7 +2435,7 @@ Scenario: Running an if request that contains a contradictory lessThanOrEqualTo
        | foo | bar |
        | 10  | 1   |
 
-@ignore
+@ignore #may need to raise bug for this
```

Comment:
Can we raise a bug or relate this issue to an existing one?

Reply:
This is being addressed in ticket #501